app/scripts/constants/contracts.js

```diff
@@ -6,3 +6,17 @@ export const SINGLE_CALL_BALANCES_ADDRESS_ROPSTEN =
   '0xb8e671734ce5c8d7dfbbea5574fa4cf39f7a54a4';
 export const SINGLE_CALL_BALANCES_ADDRESS_KOVAN =
   '0xb1d3fbb2f83aecd196f474c16ca5d9cffa0d0ffc';
+export const SINGLE_CALL_BALANCES_ADDRESS_GOERLI =
+  '0xb1d3fbb2f83aecd196f474c16ca5d9cffa0d0ffc';
```

Comment:
This address doesn't match the contract address you listed on https://github.com/wbobeirne/eth-balance-checker/pull/39, is that intentional?